### ISSUE TITLE
The Shulker box overlay for Tweakeroo/Minihud doesn't mesh well with REI

### ISSUE THREAD

**Issue by Cookieiowastaken** [shedaniel/RoughlyEnoughItems#565]
The background for Shulker box overlay is behind the REI items screen and the items are in front which makes it hard to see the items. Can this be fixed?
![Minecraft_ 1 17 2021-06-21 15-39-46_Moment](https://user-images.githubusercontent.com/86268120/122833529-5fe44200-d2aa-11eb-8864-409ac9f6cbdb.jpg)

**Comment by helgeh** [comment]
I have this problem too. Have you reported this on Tweakeroo/MiniHUD also?

**Comment by Cookieiowastaken** [comment]
Yeah I reported it to Tweakeroo/MiniHUD after I figured out where to report it. They said they know the problem and will probably put in their own fix soon. I am not sure how much of the issue is REI and how much is Tweakeroo/MiniHUD.

**Comment by helgeh** [comment]
Great! If they are aware and have a plan for solving it then I'm happy :)
